src/custom/utils/operator.ts

```diff
@@ -203,23 +205,37 @@ function toApiAddress(address: string, chainId: ChainId): string {
   return address
 }
 
-async function _getJson(chainId: ChainId, url: string): Promise<any> {
-  let response: Response | undefined
+async function _getJson<T, E>(chainId: ChainId, url: string): Promise<ErrorHandlingResponse<T, E>> {
+  const response = {
+    data: undefined,
+    error: undefined
+  }
+
   try {
     const responseMaybeOk = await _fetchGet(chainId, url)
-    response = responseMaybeOk.ok ? responseMaybeOk : undefined
-  } catch (error) {
-    // do nothing
-  }
 
-  if (!response) {
-    throw new Error('Error getting the fee')
-  } else {
-    return response.json()
+    // response may be unsupported token
+    if (!responseMaybeOk.ok && responseMaybeOk.status !== 400) {
+      // if non-ok status is not unsupported token error, throw
+      throw new Error('Error getting fee')
+    } else {
+      const data = await responseMaybeOk.json()
+      return {
+        ...response,
+        // response is good, 200, return as data
+        data: responseMaybeOk.status === 200 && data,
+        // response is not good, but is an UnsupportedToken error to use
+        error: responseMaybeOk.status === 400 && data
```

Comment:
This error handling is fragile. We shouldn't do it here, in the generic _fetch function.
What happens if the backend returns 401, 402, 40X, 202, etc.

The logic of the error handling should be where we have the information of what to do with it.